sdk/search/azure-search-documents/tests/test_queries.py

```diff
@@ -10,13 +10,13 @@
 except ImportError:
     import mock
 
-from azure.search.documents._index._generated.models import (
+from azure.search.documents._internal._generated.models import (
     AutocompleteRequest,
     SearchRequest,
     SuggestRequest,
```

Comment:
I think the original import was not ideal, these are available from public API, I think?

Reply:
No they are not. Should we expose them?

Reply:
Hrmm actually I was thinking of `AutocompleteQuery` etc, nevermind!